src/snyk/cli/services/cliService.ts

```diff
@@ -101,6 +103,11 @@ export abstract class CliService<CliResult> extends AnalysisStatusProvider {
     args.push(...foldersToTest);
     args.push('--json');
 
+    const additionalParams = this.config.getAdditionalCliParameters();
+    if (additionalParams) {
+      args.push(...additionalParams.trim().split(' '));
```

Comment:
if params holds space inside (for example valid case with `--json-file-output="g o o f.json"` ) we will fail